esiclient/v1/orchestrator/cluster.py

```diff
@@ -222,3 +223,80 @@ def take_action(self, parsed_args):
 
         return ["Node", "Port", "Network", "Fixed IP",
                 "Floating Network", "Floating IP"], data
+
+
+class Undeploy(command.Command):
+    """Undeploy a cluster from ESI nodes"""
+
+    log = logging.getLogger(__name__ + ".Undeploy")
+    REQUIRED_FIELDS = ['nodes', 'private_network_name', 'api_vip',
```

Comment:
I think REQUIRED_FIELDS is not used to check fields in config file in this class.